packages/fether-react/babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
For fether-react, we do not use any additional babel plugins, we use craco, which already bundles everything.